js/mobile-menu.js

```diff
@@ -1,21 +1,26 @@
 /* eslint-disable linebreak-style */
-const mobileMenuIcon = document.querySelector('#humber-icon');
-const mobileMenu = document.querySelector('.mobile-menu');
-const menuLink = document.querySelectorAll('.menu-link');
-const mobileMenuClose = document.querySelector('.mobile-menu-close-btn');
 
-mobileMenuIcon.addEventListener('click', (e) => {
-  e.preventDefault();
-  mobileMenu.style.display = 'flex';
-});
+// const mobileMenuIcon = document.querySelector('#humber-icon');
+// const mobileMenu = document.querySelector('.mobile-menu');
+// const menuLink = document.querySelectorAll('.menu-link');
+// const mobileMenuClose = document.querySelector('.mobile-menu-close-btn');
 
-mobileMenuClose.addEventListener('click', (e) => {
-  e.preventDefault();
-  mobileMenu.style.display = 'none';
-});
+// mobileMenuIcon.addEventListener('click', (e) => {
+//   e.preventDefault();
+//   mobileMenu.style.display = 'flex';
+// });
 
-menuLink.forEach((link) => {
-  link.addEventListener('click', () => {
-    mobileMenu.style.display = 'none';
-  });
-});
+// mobileMenuClose.addEventListener('click', (e) => {
+//   e.preventDefault();
+//   mobileMenu.style.display = 'none';
+// });
+// mobileMenuClose.addEventListener('click', (e) => {
+//   e.preventDefault();
+//   mobileMenu.style.display = 'none';
+// });
+
+// menuLink.forEach((link) => {
+//   link.addEventListener('click', () => {
+//     mobileMenu.style.display = 'none';
+//   });
+// });
```

Comment:
- [ ] Please remove this file as it's not used.  Commented code makes the project look not professional and not organized. This will make your code look much cleaner 🧼 and maintainable 😄